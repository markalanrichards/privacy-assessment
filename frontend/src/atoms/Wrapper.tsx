import * as React from "react";
import styled from "styled-components";

const Article = styled.article`
  grid-area: null;
`;
interface AtomProps {
  children: JSX.Element | string;
}

export const Wrapper: React.SFC<AtomProps> = ({ children }): JSX.Element => (
  <Article>{children}</Article>
);
