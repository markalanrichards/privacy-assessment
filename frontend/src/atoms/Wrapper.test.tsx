import * as React from "react";
import { mount } from "enzyme";

import { Wrapper } from "./Wrapper";

test("Wapper component", () => {
  const wrapper = mount(<Wrapper>Some text</Wrapper>);

  const text = wrapper.find("article").text();

  expect(text).toEqual("Some text");
  expect(wrapper).toMatchSnapshot();
});
