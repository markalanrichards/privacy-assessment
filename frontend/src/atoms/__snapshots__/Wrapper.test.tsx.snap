// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wapper component 1`] = `
<Component>
  <styled.article>
    <article
      className="sc-bdVaJa DPkqp"
    >
      Some text
    </article>
  </styled.article>
</Component>
`;
